<launch>
  <node name="so_dso_fbow" pkg="so_dso_loop_closure" type="test_fbow" output="screen">
    <param name="voc_file" value="$(find so_dso_loop_closure)/loop_closure/src/FBoW/orb_mur.fbow" />
    <param name="incoming_id_file1" value="$(find so_dso_loop_closure)/loop_closure/results/RobotCar/2015-02-10-11-58-05/incoming_id_file.txt" />
    <param name="incoming_id_file2" value="$(find so_dso_loop_closure)/loop_closure/results/RobotCar/2014-12-12-10-45-15/incoming_id_file.txt" />
    <param name="bag1" value="/home/jiawei/Workspace/data/bags/RobotCar/2015-02-10-11-58-05.bag" />
    <param name="bag2" value="/home/jiawei/Workspace/data/bags/RobotCar/2014-12-12-10-45-15.bag" />
    <param name="img_topic" value="camera/left/image_raw" />
    <param name="output_file" value="$(find so_dso_loop_closure)/loop_closure/results/RobotCar/FBoW/t10/dist_mat.txt" />
  </node>
</launch>
